### DISCUSSION TITLE
MW4 for quick modeling with other (not 10Micron) mounts?

### DISCUSSION THREAD

**Discussion by airscottdenning** [mworion/MountWizzard4#250] [Feature Request]
I recently bought a RainbowAstro RST135e mount. It uses 5 stars for a simple pointing model. 

I'd like to generate/solve/sync the 5 points automatically with MW4 if that's possible, but it appears the software is pretty closely tied to the 10Micron driver.

Is this something I could do, or that could be added to MW4 in the future?

Many thanks for all your amazing support!

**Comment by mworion** [comment]
Hi Scott,

could you provide some information about the mount? Command protocol etc. I have no clue how far the solution might be away from the 10micron solution.

Michel

**Comment by airscottdenning** [comment]
Hi Michel,

Yes, the mount has an ASCOM driver and there are two protocols: "Rainbow" (many features) and "LX200" (very standard but limited). 

For modeling I would think slew/solve/sync using LX200 protocol should be fine. In the meantime I can just slew with the handset, solve with ASTAP, and then press "Sync" on the handset.

Thank you for considering!
Scott

**Reply by mworion** [reply]
Hi Scott, how is the ASTAP solution transferred to the mount ?
Still not an easy task (not related to the modeling itself, but writing a new mount driver and changing architecture). I'm still working on v3 and I have to finish that first before having a closer look to a solution. Meanwhile we could chat about the topics.
Michel

**Comment by airscottdenning** [comment]
Just through a mount Sync command. The ASCOM driver has a toggle where Sync does or does not update the pointing model.
